pkg/deploy/deploy.go

```diff
@@ -271,7 +271,11 @@ func deployConfig(ctx context.Context, c *config.Config, clients ClientSet, reso
 	var deployErr error
 	switch c.Type.(type) {
 	case config.SettingsType:
-		resolvedEntity, deployErr = setting.Deploy(ctx, clients.Settings, properties, renderedConfig, c)
+		var insertAfter string
+		if ia, ok := properties["insertAfter"]; ok {
```

Comment:
nit: I'd make this a fixed constant param name like name and scope are. IIRC defined in either the config or parameter package